helm-chart/openj9-jitserver-chart/templates/service.yaml

```diff
@@ -0,0 +1,37 @@
+{{- /*******************************************************************************
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made available under
+* the terms of the Eclipse Public License 2.0 which accompanies this
+* distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+* or the Apache License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+* 
+* This Source Code may also be made available under the following
+* Secondary Licenses when the conditions for such availability set
+* forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+* General Public License, version 2 with the GNU Classpath
+* Exception [1] and GNU General Public License, version 2 with the
+* OpenJDK Assembly Exception [2].
+* 
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+* 
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+*******************************************************************************/ -}}
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "openj9-jitserver-chart.fullname" . }}
+  labels:
+    {{- include "openj9-jitserver-chart.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
We can also add under ``type: {{ .Values.service.type }}``,  ``sessionAffinity: {{ .Values.service.sessionAffinity }}``. This will ensure that connections from a particular client are passed to the same pod each time. Since a  single client should not be allowed to connect to multiple JITServer instances.